src/Quotes/Quote.js

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+
+export function Quote() {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  const involve = {
+    headers: { 'X-Api-Key': '8aT24gv6TBZiwqX+BHiGug==VwbWI4RgUoRH1xu0' },
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const res = await fetch('https://api.api-ninjas.com/v1/quotes?category=forgiveness', involve);
+
+        const json = await res.json();
+        setData(json[0]);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setData, setIsLoading]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
- [ ] In this [review link](https://github.com/samuelkinyanjui/mathmagicianone/pull/4#discussion_r1306383105), the previous reviewer requested that you should not disable eslint, however you re yet to apply this change kindly do so 👍

Reply:
Dear Efiamotu this was optional, not mandatory